CHANGELOG.md

```diff
@@ -11,8 +11,43 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Added hooks to metric module interface ([#2528](https://github.com/PyTorchLightning/pytorch-lightning/pull/2528/))
 
+- Added warn user when dropping unpicklable `hparams` ([#2874](https://github.com/PyTorchLightning/pytorch-lightning/pull/2874))
+
 ### Changed
 
+- Refactored accelerator backends:
```

Comment:
I don't think we should include these internal refactors in changelog. They don't change the functionality and are invisible to the user.

Reply:
Well, I would at least include numbers, but even if it changes internal flow it may be source of confusion as hooks are called from different places...

Reply:
I think it is a good idea to keep a backlog of all the refactors happening. Maybe a compromise would be to add a new section called `Refactors`?

Reply:
cc: @PyTorchLightning/core-contributors

Reply:
I like the idea of having a refactor section for the next 1 or 2 releases